dotnet/message-service/Controllers/MessageController.cs

```diff
@@ -29,28 +22,26 @@ public MessageController(Tracer tracer, IConnectionMultiplexer redisConnection)
         [HttpGet]
         public async Task<string> GetAsync()
         {
-            using var span = _tracer.StartActiveSpan("Getting Message");
-            {
-                span.SetAttribute("testAttribute", "Message");
-                var message = await DetermineMessage();
-                return message;
-            }
-        }
 
-        private async Task<string> DetermineMessage()
-        {
-            var db = _redisConnection.GetDatabase();
-            var rng = new Random();
-            var i = rng.Next(0, 9);
-            db.StringSet("message", Messages[i]);
-
-            await SleepAwhile();
-            return db.StringGet("message");
+            var message = await DetermineMessage();
+            return message;
         }
 
-        private static async Task SleepAwhile()
+        private async Task<string> DetermineMessage()
         {
-            await Task.Delay(50);
+            using var span = _tracer.StartActiveSpan("📖 look up message ✨");

```

Comment:
Moved the span into this method and gave it the fancy emojified version used in the other languages to help make our manual instrumentation spans stand out visually from the spans generated by auto-instrumentation.